var/spack/repos/builtin/packages/vasp/package.py

```diff
@@ -227,7 +298,7 @@ def build(self, spec, prefix):
         if "+cuda" in self.spec:
             make("gpu", "gpu_ncl")
         else:
-            make("std", "gam", "ncl")
+            make("std")  # , "gam", "ncl")
```

Comment:
Why did you disable builds of the gamma and non-collinear versions?

Reply:
This has been fixed now!